asset/decompress/zip.go

```
@@ -0,0 +1,66 @@
+
+	// Start async processing
+	go func() {
+		if err := d.processZipFile(ctx, zipReader); err != nil {
+			// Update status to ERROR if processing fails
+			d.assetService.Update(ctx, assetID, asset.UpdateAssetInput{
+				Status: asset.StatusError,
+				Error:  err.Error(),
+			})
+		} else {
+			// Update status to ACTIVE if processing succeeds
+			d.assetService.Update(ctx, assetID, asset.UpdateAssetInput{
```

golangci-lint (1.62.2)

Error return value of `d.assetService.Update` is not checked

(errcheck)